### PR TITLE
WPB-5204 Removing unused code

### PR DESCRIPTION
**Pull request by lepsa**
Removing code around APNS_VOIP

## Checklist

 - [:heavy_check_mark:] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [:heavy_check_mark:] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

File: changelog.d/5-internal/WPB-5204 [hunk 1]
```diff
@@ -0,0 +1 @@
+Removed APNS_VOIP code.
```
Comment (supersven) [inline changelog.d/5-internal/WPB-5204:1]:
```suggestion
Removed APNS_VOIP code. (APNS_VOIP is a native push notification channel which we aren't using anymore.)
```

File: libs/wire-api/test/golden/Test/Wire/API/Golden/Manual/Token.hs [hunk 2]
```diff
@@ -1,29 +0,0 @@
...
-
-module Test.Wire.API.Golden.Manual.Token where
-
-import Data.Id
-import Wire.API.Push.V2.Token
-
-testObject_Token_1 :: PushToken
-testObject_Token_1 =
-  pushToken
-    APNSVoIPSandbox
```
Comment (supersven) [inline libs/wire-api/test/golden/Test/Wire/API/Golden/Manual/Token.hs:26]: Wouldn't it be better to keep this test and only adjust the `Transport` (e.g. use `APNSSandbox`)? I.e. are you sure that `PushToken`'s serialization is still sufficiently tested?

Comment (supersven) [reply libs/wire-api/test/golden/Test/Wire/API/Golden/Manual/Token.hs:26]: Ah, never mind. `testObject_PushToken_user_1` does this. :sweat_smile: 

```haskell
testObject_PushToken_user_1 :: PushToken
testObject_PushToken_user_1 =
  pushToken
    APNSSandbox
    (AppName {appNameText = "G\1008289"})
    (Token {tokenText = "\EOT8M\NAKAv\1104873"})
    (ClientId {client = "16"})
```

File: libs/ropes/src/Ropes/Twilio.hs [hunk 3]
```diff
@@ -129,7 +129,6 @@ data CarrierInfo = CarrierInfo
 data PhoneType
   = Landline
   | Mobile
-  | VoIp
```
Comment (supersven) [inline libs/ropes/src/Ropes/Twilio.hs:132]: I'm unfortunately unsure how twilio relates to our SNS sending code :thinking:

Comment (flokli) [reply libs/ropes/src/Ropes/Twilio.hs:132]: This seems to be entirely unrelated - the Twilio `PhoneType` is something entirely different, this PR shouldn't be touching this.

**Review by supersven — APPROVED** [review]
After I reverted the Twilio-related changes, this show be good now :+1:

**Pull request merged**